cms/search/tests/test_api_views.py

```diff
@@ -80,8 +81,22 @@ def test_resources_available_in_external_env(self):
         response = self.call_view_as_external(RESOURCE_ENDPOINT)
         self.assertEqual(response.status_code, 200)
 
+    @override_settings(CMS_RESOURCES_ENDPOINT_ENABLED=False)
+    def test_resources_endpoint_disabled_returns_404(self):
+        """When the CMS_RESOURCES_ENDPOINT_ENABLED setting is False, the endpoint should return 404."""
+        response = self.client.get(RESOURCE_ENDPOINT)
+        self.assertEqual(response.status_code, 404)
+
+    @override_settings(CMS_RESOURCES_ENDPOINT_ENABLED=False)
+    @override_settings(IS_EXTERNAL_ENV=False)
```

Comment:
We don't need to override the `IS_EXTERNAL_ENV=False` here as we already override this on the class level.

Reply:
Thanks, this was actually supposed to be `True`, which I now changed.

Reply:
Make more sense now. Resolving!